tests/ui/zero_sized_map_values.rs

```diff
@@ -0,0 +1,45 @@
+#![warn(clippy::zero_sized_map_values)]
```

Comment:
Let's add a couple more tests using
* `static`
* `const`
* type alias
* `enum`
* a method

Reply:
Added tests for these items